src/gen_rpc_helper.erl

```diff
@@ -41,11 +43,47 @@
          get_control_receive_timeout/0,
          get_inactivity_timeout/1,
          get_async_call_inactivity_timeout/0,
-         get_listen_ip_config/0]).
+         get_listen_ip_config/0,
+         term_to_iovec/1]).
 
 %%% ===================================================
 %%% Public API
 %%% ===================================================
+
+%% term_to_iovec/1 wrapper to conditionally compress based on threshold
+-spec term_to_iovec(term()) -> erlang:iovec().
+term_to_iovec(Term) ->
+  {ok, Compress} = application:get_env(?APP, compress),
+  do_term_to_iovec(Term, Compress).
+
+do_term_to_iovec(Term, Compress) when is_integer(Compress), Compress >= 1 ->
```

Comment:
Please move this function to "Private functions" section.

Reply:
Done! https://github.com/emqx/gen_rpc/pull/54/commits/42416569ae333b2b243e60cad038f63cab984c1a